non-oss/pharos_pro/addons/kontena-storage/addon.rb

```diff
@@ -122,24 +134,47 @@ def set_defaults
     }
   end
 
+  # @return [String]
+  def rook_version
+    self.class.version.split('+').first
+  end
+
+  # @param version [String]
+  # @return [Boolean]
+  def upgrade_from?(version)
+    operator = kube_client.api('apps/v1beta1')
+                .resource('deployments', namespace: 'kontena-storage-system')
+                .get('kontena-storage-operator')
+
+    operator.spec.template.spec.containers.first.image.include?("rook-ceph:v#{version}")
+  rescue K8s::Error::NotFound
+    false
+  end
+
   # @return [K8s::Resource]
   def build_cluster_resource
     K8s::Resource.new(
-      apiVersion: 'ceph.rook.io/v1beta1',
-      kind: 'Cluster',
+      apiVersion: 'ceph.rook.io/v1',
+      kind: 'CephCluster',
       metadata: {
         name: 'kontena-storage',
         namespace: 'kontena-storage'
       },
       spec: {
+        cephVersion: {
+          image: 'docker.io/ceph/ceph:v13.2.4-20190109'
```

Comment:
Not 💯 sure about this version... and we should mirror the image to pharos repository.